pywwt/layers.py

```diff
@@ -0,0 +1,100 @@
+import re
+
+class ImageryLayers():
+
+    def __init__(self, layer_list):
+        self._layers = {}
+        self._spectrum = ['gamma', 'x', 'uv', 'visible', 'ir', 'micro', 'radio', 'other']
+        for band in self._spectrum:
+            self._layers[band] = {}
+        
+        self._list2dict(layer_list)
+
+    def _list2dict(self, og_list):
+        for layer in og_list:
+            ind = 0
+
+            if(re.search(r'(?i)gamma',layer) != None):
```

Comment:
The parentheses around the whole if expression are not needed - also for ``None`` one normally writes ``is not None`` (same comment for if statements below)